plugin.php

```diff
@@ -17,13 +17,15 @@
 	exit;
 }
 
+define('ALPS_GUTENBERG_VERSION', '1.5.1');
```

Comment:
Can this version number be pulled from the `kernl.version` file as part of the `npm build` script?